jsapp/js/account/stripe.api.ts

```diff
@@ -48,13 +50,62 @@ export async function changeSubscription(
   });
 }
 
-export const useOrganizationQuery = () => useQuery({
-  queryFn: async () => {
-    const response = await fetchGet<PaginatedResponse<Organization>>(endpoints.ORGANIZATION_URL);
-    return response.results?.[0];
-  },
-  queryKey: [QueryKeys.organization],
-});
+export const useOrganizationQuery = () => {
+  const isMmoEnabled = useFeatureFlag(FeatureFlag.mmosEnabled);
+
+  // Using a separated function to fetch the organization data to prevent
+  // feature flag dependencies from being added to the hook
+  const fetchOrganization = async (): Promise<Organization> => {
+    // Once we have the full data from backend, this function should be modified to return
+    // the actual data.
+
+    if (isMmoEnabled) {
+
+      const currentAccount = sessionStore.currentAccount as AccountResponse;
+
+      if (!currentAccount) {throw new Error('No account data found');}
+
+        const organizationUrl = currentAccount.organization?.url;
+
+        // Here we would fetch organization data from organizationSlug
+        // For now, we will return mocked data
+        console.log('Fetching organization data from:', organizationUrl);
+
+        // Return mocked data for MMO
+        return {
```

Comment:
I think in the past we added some artificial delay to make it more "realistic", what do you think about that?

Reply:
Good one. Just added it.